src/components/Checkbox/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BaseCheckbox from './BaseCheckbox';
+
+const Checkbox = props => (
+    <BaseCheckbox
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        onMouseDown={props.onMouseDown ? props.onMouseDown : props.onPress}
+        onPress={undefined}
```

Comment:
```suggestion
```

Reply:
Actually this is important to set it as undefined because not doing so makes this event fire twice. That was the reason it was causing a regression earlier.